whitelabelco/sofo-foods
# Cp/feat/remove footer links

### Description 💻

Resolves: #181 

### Implementation 💡

Removes A&M cheese and Great Lakes Box Company links from footer

[Comps](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=1-4&node-type=frame&t=rpWkjn7sRlrBYqzF-0)

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- scroll down to footer and confirm the above links are removed

### Screenshots/Demos 📷

### Checklist ✅
